events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -0,0 +1,747 @@
+//  triviaMasterClientStandard.js
+//
+//  Created by Rebecca Stankus on 06/11/18
+//  Copyright 2018 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+//  This is the standard version of "Last Avatar Standing" Trivia 
+//  that draws from a third-party database of thousands of questions.
+
+/* global EventBridge Users AccountServices */
+
+(function() {
+
+    var SECRETS = Script.require(Script.resolvePath('triviaInfo.json')),
+        TRIVIA_CHANNEL = SECRETS.TRIVIA_CHANNEL,
+        GOOGLE_URL = SECRETS.GOOGLE_URL,
+        TABLET_BUTTON_IMAGE = Script.resolvePath('../entities/icons/questionMark-i.png'),
+        TABLET_BUTTON_PRESSED = Script.resolvePath('../entities/icons/questionMark-a.png'),
+        SEARCH_RADIUS = 100,
+        ONE_SECOND_MS = 1000,
+        TEN_SECONDS_MS = 10000,
+        ZONE_COLOR_INDEX = 19,
+        HALF_MULTIPLIER = 0.5,
+        FIRST_WAIT_TO_COUNT_AVATARS = 1500,
+        WAIT_TO_SHOW_QUESTION = 500,
+        MIN_PLAYERS = 3,
+        HFC_INCREMENT = 100,
+        HFC_HALVER = 0.5,
+        MIN_PRIZE = 300,
+        HOST_PERCENTAGE = 0.1;
+
+    var tablet = Tablet.getTablet('com.highfidelity.interface.tablet.system'),
```

Comment:
`appUI` would render these things unnecessary - don't change it this time around though :)

Reply:
yeah, seemed like a sizeable overhaul...